src/ports.js

```diff
@@ -31,6 +31,24 @@
             this.ports.push(port);
         },
 
+        storeLayoutPosition: function (id, position) {
```

Comment:
`set` seems to me that it could affect the port position, in this case value is just stored for later use, port position is not affected.